test/run_workflows/skipped_steps_test.exs

```diff
@@ -20,10 +19,16 @@ defmodule StepFlow.RunWorkflows.SkippedStepsTest do
       version_minor: 5,
       version_micro: 4,
       reference: "some id",
+      icon: "custom_icon",
+      label: "Simple workflow test",
+      tags: ["test"],
+      parameters: [],
```

Comment:
Is it skipped because no parameters are given?

Reply:
it's skipped as no `source_paths` is provided